docs/versioned/serving/config-network-adapters.md

```diff
@@ -0,0 +1,187 @@
+---
+audience: administrator
+components:
+  - serving
+function: how-to
+---
+
+# Configure Knative networking
+
+This page provides installation and configuration guidance for Knative networking. You can configure Ingress controls, service-meshes, and gateways.
+
+### Determine current state
+
+Use the following command to determine which ingress controllers are installed and their status.
+
+```bash
+kubectl get pods -n knative-serving
+```
+
+The ingress controllers, that have been tested for Knative, have the following base names:
+
+- Kourier: `kourier-control-*`, and `kourier-gateway-*`. Kourier is included in the Knative Serving installation should appear in the results when your cluster is first created.
+- Contour: `contour-*`
+- Istio: `istio-webhook-*`. The main Istio control plane pods such as `istiod-*` are in the `istio-system` namespace. Knative adds the `istio-webhook-*` pod in the `knative-serving` namespace when Istio is the chosen networking layer.
+
+The `network-config` ConfigMap sets which controller to use with the ingress controller key. This key is patched with the name of any new controller. See [Changing the ingress controller](#change-the-controller) for important information about using this key.
```

Comment:
This is actually slightly different -- each ingress controller only manages ingress objects that are annotated with its key.  Knative Serving uses a default value of the key based on the `network-config` ConfigMap.  So this is somewhat of a cooperative process between the ingress controller and the core Knative controller.

Reply:
(The end result is effectively the same, but the mechanism is different, which seemed important to correct in the documentation.)